docs/cli.md

```diff
@@ -1082,6 +1082,65 @@ The `whoami` command returns the Viam user for an authenticated CLI session, or
 viam whoami
 ```
 
+### `auth-app`
+
+The `auth-app` command allows you to register and update your mobile or web app (created with the Viam Flutter or TypeScript [SDKs](/sdks/)) with [FusionAuth](https://fusionauth.io/) (the tool Viam uses for authentication and authorization) so that you or other users can log into your app with the same credentials they use to log into the [Viam app](https://app.viam.com).
+The user's credentials allow them the same [permissions](/fleet/rbac/) to organizations, locations, and machines that they have in the Viam app.
+
+Examples:
+
+```sh {class="command-line" data-prompt="$"}
+viam auth-app register --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-name="julias app" --origin-uris="https://test.com","https://test2.com" --redirect-uris="https://redirect-url.com" --logout-uri="https://logout.com"
+Info: Successfully registered auth application
+{
+  "application_id": "1234a1z9-ab2c-1234-5678-bcd12345678a",
+  "application_name": "julias app",
+  "secret": "supersupersecretsecret"
+}
+
+viam auth-app update --org-id=z1234567-1a23-45a6-a11b-abcdefg1234 --application-id=1234a1z9-ab2c-1234-5678-bcd12345678a --redirect-uris="https://test.com","https://test2.com"
+Info: Successfully updated auth application
+{
+  "application_id": "1234a1z9-ab2c-1234-5678-bcd12345678a",
+  "application_name": "julias app"
+}
+```
+
+{{% alert title="Caution" color="caution" %}}
+Do not share the secret returned by `auth-app register` publicly.
+Sharing this information could compromise your system security by allowing unauthorized access to your machines, or to the computer running your machine.
+{{% /alert %}}
+
+#### Command options
+
+| Command option | Description                             |
+| -------------- | --------------------------------------- |
+| `register`     | Register an application with FusionAuth |
+| `update`       | Update your application                 |
+
+##### Named arguments: `register`
+
+<!-- prettier-ignore -->
+| Argument             | Description                                                       | Inclusion    |
+| -------------------- | ----------------------------------------------------------------- | ------------ |
+| `--org-id`           | The {{< glossary_tooltip term_id="organization" text="organization" >}} ID to with which to associate this app | **Required** |
+| `--application-name` | A name of your choosing for your application                      | **Required** |
```

Comment:
(opt)
```suggestion
| `--application-name` | The name of your application                      | **Required** |
```
or something like `Your name for your application` -- just think a little awkward sounding

Reply:
I want to make it clear that this is just an arbitrary identifier. Is this any better?
```suggestion
| `--application-name` | A name (of your choice) to identify your application | **Required** |
```

Reply:
I assume you might be able to register Oauth for multiple applications and you'd want to be able to differentiate. So maybe "The name of your application which will be used to identify the application". Pending Clint's input

Reply:
Actually we don't need to wait on Clint here, you can see where it is used in the update command and you've explained there that it's needed to identify the app. 

```suggestion
| `--application-name` | The name of your application which is used to identify the application | **Required** |
```

Reply:
He said the other day that app name is arbitrary, so idk *where* it's used to identify the application

Reply:
I think saying it's used to identify the application is enough. However, @ohEmily might also know whether it's used in FusionAuth or on our side of things

Reply:
it is passed along to FusionAuth, yes. FusionAuth's string is: "The name of the Application. This value is for display purposes only and can be changed at any time." ([docs](https://fusionauth.io/docs/get-started/core-concepts/applications)) in case that's helpful!

Reply:
Thank you @ohEmily ! - @JessamyT feel free to link to the fusionauth docs

Reply:
```suggestion
| `--application-name` | A display name (of your choice) for your application. | **Required** |
```